softdev.bib

```diff
@@ -3333,3 +3333,10 @@ @article{yanovski21ghostcell
   year={2021},
   month = Aug,
 }
+
+@misc{cheng20typefreezing,
+    title = "Type freezing: exploiting attribute type monomorphism in tracing JIT compilers",
+    author = "Cheng, Lin and Ilbeyi, Berkin and Bolz-Tereick, Carl Friedrich and Batten, Christopher",
+    year = 2020,
```

Comment:
We also ask for a month if at all possible.

Reply:
Month added :point_right: 3886dfe9e36eb83a454e9f3ecd7b39135f8191d5